cli/src/wallet.rs

```diff
@@ -73,43 +260,58 @@ pub async fn new_wallet(cli: WalletCli) -> Result<(Option<Wallet>, Option<Accoun
                 mnemonic_command(output_file_name, output_stdout).await?;
                 return Ok((None, None));
             }
-            WalletCommand::NodeInfo => {
-                node_info_command(storage_path).await?;
-                return Ok((None, None));
-            }
         }
     } else {
-        // no command provided, i.e. `> ./wallet`
-        match (storage_path.exists(), snapshot_path.exists()) {
-            (true, true) => {
-                let password = get_password("Stronghold password", false)?;
-                let wallet = unlock_wallet(storage_path, snapshot_path, password).await?;
-                if wallet.get_accounts().await?.is_empty() {
-                    create_initial_account(wallet).await?
-                } else if let Some(alias) = cli.account {
-                    (Some(wallet), Some(alias))
-                } else if let Some(account) = pick_account(&wallet).await? {
-                    (Some(wallet), Some(account.alias().await.into()))
-                } else {
-                    (Some(wallet), None)
+        // no wallet command provided
+        if let Some((wallet, linked_secret_manager)) = wallet_and_secret_manager {
+            match linked_secret_manager {
+                LinkedSecretManager::Stronghold {
+                    snapshot_exists: None,
+                    snapshot_path,
+                } => {
+                    println_log_error!(
+                        "Snapshot file for Stronghold secret manager linked with the wallet not found at '{}'",
+                        snapshot_path.display()
+                    );
+                    return Ok((None, None));
+                }
+                LinkedSecretManager::LedgerNano { is_connected: false } => {
+                    println_log_error!("Ledger Nano linked with the wallet not connected");
+                    return Ok((None, None));
```

Comment:
That's annoying, why not still allow to open the wallet? To check balance or see the txs it's not necessary to have a ledger nano connected

Reply:
yeah, it's annoying, but if we do it lazily (only check that it's connected), then we should do that as well for Stronghold? Only ask for the password for operations that require it?

Reply:
should we make it a separate issue (and PR) that deals with removing that annoyance (because this PR is just concerned with Ledger Nano support)?

Reply:
I think it's a bit different here because you don't have to enter the password at a later point (if you just connect a ledger before doing a tx or creating a new address it will not fail, for stronghold it would without password), so you just have to not add this extra match?
If it's needed for whatever reason I don't see, feel free to close and for stronghold you can open an issue if you want, definitely not needed in this PR